pkg/helm/values.go

```diff
@@ -0,0 +1,55 @@
+package helm
+
+import (
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"net/url"
+	"os"
+
+	"k8s.io/apimachinery/pkg/util/yaml"
+)
+
+func LoadChartValuesOverrides(chartValues *map[string]interface{}, paths []string) (map[string]interface{}, error) {
+	var err error
+
+	valuesOverride := make(map[string]interface{})
+
+	for _, path := range paths {
+		var data []byte
+		if data, err = readValuesOverride(path); err != nil {
+			return nil, err
+		}
+
+		if err = yaml.Unmarshal(data, &valuesOverride); err != nil {
+			return nil, err
+		}
+
+		if err = yaml.Unmarshal(data, chartValues); err != nil {
+			return nil, err
+		}
+	}
+
+	return valuesOverride, nil
+}
+
+func readValuesOverride(path string) ([]byte, error) {
```

Comment:
is this really needed ? like a preparation for fetching presets from our docs or something?